### ISSUE TITLE
Page title not marked as a heading

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1621]
The title ‘Settings’, which appears in the top navigation bar, is not marked up as a heading. This is a problem, since headings can help screen reader users understand the hierarchy of content on the page. To resolve this, the ‘Settings’ title should be correctly marked up using <h1>
